woo-razorpay.php

```diff
@@ -2215,6 +2215,11 @@ public function update1ccOrderWC(& $order, $wcOrderId, $razorpayPaymentId)
                         // $item->set_method_id( "flat_rate:1" );
                         $item->set_total( $razorpayData['shipping_fee']/100 );
 
+                        if (isset($razorpayData['shipping_method']))
```

Comment:
this change is in else block, when will the if block execute? 
there also we are adding WC_Order_Item_shipping

Reply:
the if block execute for wcfm plugins shipping for woo Commerce, doesn't think that this changes should be added there too. will discuss with @ChetanGN on the same.

Reply:
please check with chetan and refactor your condition to `isset($razorpayData['shipping_method']) &&  isset($razorpayData['shipping_method']['name'])`

Reply:
also we need to set shipping method name for 0 shipping fee as well